includes/template-library/manager.php

```diff
@@ -54,6 +56,8 @@ class Manager {
 	 * @access public
 	 */
 	public function __construct() {
+		Data_Manager::instance()->register_controller( Controller::class );
```

Comment:
Should we use `Plugin::$instance->data_manager` or this?

Reply:
Handled.